filters/laser_similarity.json

```diff
@@ -27,5 +21,5 @@
             "help": "Two-letter target language code (ISO 639-1)"
         }
     },
-    "command": "./laser_similarity.py --threshold $THRESHOLD --batch-size $BATCHSIZE --src-lang $SRCLANG --tgt-lang $TGTLANG"
+    "command": "../threshold.py --cache $TMPDIR/laser.$(echo \"$SRCLANG $TGTLANG\" | cksum | cut -d' ' -f1).dbm $THRESHOLD ./laser_similarity.py --scores --src-lang $SRCLANG --tgt-lang $TGTLANG"
```

Comment:
How does this run? I see no `__main__()` or anything? Where does python default to?